www/scripts/codecheckerviewer/BugViewer.js

```diff
@@ -980,17 +973,14 @@ function (declare, domClass, dom, style, fx, Toggler, on, query, Memory,
 
       //--- Show or hide review status message box ---//
 
-      this.showOrHideReviewStatusMessageBox(report.review.comment);
+      this.showReviewStatusMessageBox(report.review.comment);
     },
 
     /**
      * Show or hide the review status message box if the review status is
      * unreviewed.
      */
-    showOrHideReviewStatusMessageBox : function (message) {
-      if (this.reportData.review.status !== ReviewStatus.UNREVIEWED)
-        domClass.remove(this._reviewComment, 'hide');
-      else
+    showReviewStatusMessageBox : function (message) {
```

Comment:
Instead of `domClass.add()` and `domClass.remove()`, it can be simplified into a [`domClass.toggle()`](https://dojotoolkit.org/reference-guide/1.7/dojo/toggleClass.html#dojo-toggleclass)

```js
showOrHideReviewStatusMessageBox : function (message) {
  domClass.toggle(this._reviewComment, 'hide', !this.reportData.review.author);
}
```

(The negation is needed because we want to **not** hide if author exists.)

Toggle works in a way that if a 3rd parameter is given, if will add or remove based on this boolean value. (So if it is already there and we want to add it again, it won't). Without this argument, it would actually work as a _toggle_.